src/qtiItem/core/interactions/CustomInteraction.js

```diff
@@ -129,6 +130,9 @@ var CustomInteraction = Interaction.extend({
             }
         });
         return this;
+    },
+    getNormalMaximum: function getNormalMaximum() {
```

Comment:
**chore:** You can use object shorthand as ES6+ is supported in this package.

**suggestion:**
```suggestion
    getNormalMaximum() {
```